sdk/storage/azblob/README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Introduction
 
-The Microsoft Azure Storage SDK for Go allows you to build applications that takes advantage of Azure's scalable cloud storage. 
+The Microsoft Azure Storage SDK for Go allows you to build applications that takes advantage of Azure's scalable cloud storage.
 This SDK replaces the previously previewed [azblob package](https://github.com/azure/azure-storage-blob-go).
```

Comment:
We should use beta/stable to refer to our libraries and preview/GA to refer to our services.
```suggestion
This is the new beta client library for Azure Blob Storage, which follows our [Azure SDK Design Guidelines for Go](https://azure.github.io/azure-sdk/golang_introduction.html) and replaces the previous beta [azblob package](https://github.com/azure/azure-storage-blob-go).
```